scipy/stats/tests/test_hypotests.py

```diff
@@ -166,11 +167,17 @@ class TestCvm_2samp(object):
     def test_invalid_input(self):
         x = np.arange(10).reshape((2, 5))
         y = np.arange(5)
-        assert_raises(ValueError, cramervonmises_2samp, x, y)
-        assert_raises(ValueError, cramervonmises_2samp, y, x)
-        assert_raises(ValueError, cramervonmises_2samp, [], y)
-        assert_raises(ValueError, cramervonmises_2samp, y, [1])
-        assert_raises(ValueError, cramervonmises_2samp, y, y, 'wrong_method')
+        msg = 'The samples must be one-dimensional'
+        with pytest.raises(ValueError, match=msg):
+            cramervonmises_2samp(x, y)
+            cramervonmises_2samp(y, x)
+        msg = 'x and y must contain at least two observations.'
+        with pytest.raises(ValueError, match=msg):
+            cramervonmises_2samp([], y)
+            cramervonmises_2samp(y, [1])
```

Comment:
Codecov / codecov/patch
scipy/stats/tests/test_hypotests.py#L177
Added line #L177 was not covered by tests

looks like I need `with ...` for every test case...

Reply:
Ah good call.
I think there is a private function like `assert_raises` that accepts `match` argument floating around somewhere...